spec/models/spree/gateway/amazon_spec.rb

```diff
@@ -38,16 +51,148 @@
   end
 
   describe '#authorize' do
-    it "succeeds" do
-      response = build_mws_auth_response(state: 'Open', total: order.total)
-      expect(mws).to(
-        receive(:authorize).
-          with(/^#{payment.number}-\w+$/, order.total/100.0, "USD").
-          and_return(response)
+    def stub_auth_request(expected_body: nil, return_values:)
+      stub_request(
+        :post,
+        'https://mws.amazonservices.com/OffAmazonPayments_Sandbox/2013-01-01',
+      ).with(
+        body: expected_body || hash_including(
+          'Action' => 'Authorize',
+          'AmazonOrderReferenceId' => 'REFERENCE',
+          'AuthorizationAmount.Amount' => '1.1',
+          'AuthorizationAmount.CurrencyCode' => order.currency
+        )
+      ).to_return(
+        return_values,
```

Comment:
Personally I don't think so. I'm a huge fan of trailing commas.  I tried putting this on the same line above but it looked weird to me. I've changed this code also at this point now. Lmk if you think it's a big deal you'd like to revisit though.